generated/nidigital/nidigital/__init__.py

```diff
@@ -11,6 +11,8 @@
 
 from nidigital.history_ram_cycle_information import HistoryRAMCycleInformation  # noqa: F401
 
+from nidigital.capture_waveform import CaptureWaveform  # noqa: F401
```

Comment:
Something I've noticed is that the `__module__` and `__qualname__` of the re-exported types is exactly what we expect logically. But might not be what the client expects.

```py
>>> from nigitial import CaptureWaveform
>>> repr(CaptureWaveform(...))
nidigital.capture_waveform.CaptureWaveform(...)
```

I've been playing around with setting the class's `__module__` when re-exporting in `__init__.py`.
This has the added bonus of "using" the class so no more suppressing `F401`.

I haven't found anything online about others doing this or pros/cons, so just some food for thought 👍

Reply:
This comment is very timely - Jay created an [issue](https://github.com/ni/nimi-python/issues/1426) just yesterday about using `__module__` in `__repr__`.

Regarding overriding class' `__module__`, I found this SO post warning against doing it: https://stackoverflow.com/a/10194338

We should look into it more.

Reply:
If you look at this through the lens of "what am I defining my API to be" it makes a lot of sense to re-define it.
1. We tell clients to `from nigitial import CaptureWaveform`, guaranteeing that this will work (up until we make an API breaking change). It logically follows the `__module__` for that type would be `nidigital`.
2. We would avoid exposing an implementation detail. If we moved where the class was defined are we sure we aren't going to break someone? Someone JSON-ifying the data, or pickling it? Setting `__module__` to `nidgitial` means we can move the class as much as we want without worrying about exposing implementation details.

It's also worth pointing out that the SO post was about dynamically generated functions/classes.

Lastly, the new way leads to shorter strings, and who doesn't love that?

Anyways, just food for thought.

Reply:
Created an issue for this - #1433